### PR TITLE
ST-6 Implemented automated testing to retrieve all route suggestions

### PR DESCRIPTION
**Pull request by ClaudiaLapalme**
This pull request produces a test that checks the route generation process. 

It goes from the home page to the routes page. Then, it enters a set to and from destination and prompts the system for directions. Finally, it makes sure that the system has produced the correct number of suggestions per method of transport

**It is to be noted that all of the tests are saved into a single .SIDE file. This file is the project that should be opened in the Selenium IDE extension for Google Chrome. Within this file, each tested GUI component will have its own test suite with its own tests.**

### CODE REVIEW

File: src/gui-tests/gui-tests.side [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+  }],
+  "suites": [{
+    "id": "b69f1641-7df7-46a5-837d-efbbd2f7c13f",
+    "name": "routes",
+    "persistSession": false,
+    "parallel": false,
+    "timeout": 300,
+    "tests": ["ae3e7da7-fc32-4c85-bef3-c6554a6660a9"]
+  }],
+  "urls": ["http://localhost:8100/"],
```
Comment (Aleoris) [inline src/gui-tests/gui-tests.side]: Dumb question... Is it safe to put the same port as the ionic serve?

Comment (ClaudiaLapalme) [reply src/gui-tests/gui-tests.side]: It's not hosted online. This is the URL that is being opened to be tested. 

ie `http://localhost:8100/` is the webpage that the test opens and starts testing on

Comment (Aleoris) [reply src/gui-tests/gui-tests.side]: Good

**Comment by danseremet** [comment]
I pulled and had a look at the .side test using the Selenium extension. It looks good to me. Just some minor questions I have:
- Should we rename the side file by what its testing e.g. `search-for-routes.side` or is this file including all system tests? (if yes, the name is good)
- Should we add the steps to as to how we can run the system tests to the readme?

**Pull request merged**